source/_components/ptvsd.markdown

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
perhaps use the Visual Studio Code logo?

Reply:
Done